Update `YOLO` description to be more specific (#403)
Schweinepriester/github-profile-achievements: README.md

||| <!-- this empty row is intentional to separate -->
Galaxy Brain | ![Galaxy Brain Badge](/images/galaxy-brain-default.png) | ✔️ | 2 accepted answers
Pull Shark | ![Pull Shark Badge](/images/pull-shark-default.png) | ✔️ | 2 pull requests merged

> YOLO | ![YOLO Badge](/images/yolo-default.png) | ✔️ | Merged a pull request on repository with multiple contributors without code review

Reviewer: as suggested by akira
```suggestion
YOLO | ![YOLO Badge](/images/yolo-default.png) | ✔️ | Merged a pull request with reviewers requested but before any reviews
```

Reviewer 2: I would suggest 
`Merged a pull request with at least one pending reviewer`
